File: Solutions/002/Solution.java
public class Solution{
    public static void main(String[] args){
        int evenSum = 0, newValue = 0, x = 1, y = 1;
        while(newValue <= 4000000){
            newValue = x + y;
            if(newValue % 2 == 0){evenSum += newValue;}
            y = x;
            x = newValue;
        }
        System.out.println("Ans: " + evenSum);
    }
}

File: Solutions/002/Solution.py
if __name__ == "__main__":
    evenSum, newValue, x, y = 0, 0, 1, 1
    while newValue <= 4000000:
        newValue = x + y
        if newValue % 2 == 0:
            evenSum += newValue
        y = x
        x = newValue
    print("Ans: " + str(evenSum))